examples/misc/01_ader_fd.ipynb

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
<p>I would recommend either defining the state variables or referencing previous tutorials appropriately. Unless this tutorial is only intended for readers who are familiar with seismic modelling, make it as simple for target audience as is reasonably possible - that is, make sure everything is clearly defined.</p><p> </p><p>I'm not a big fan of the phrasing here:</p><p>"one can derive expressions for these higher-order time derivatives in terms of spatial derivatives. High-order..."</p><p>Just say what you're going to do in the tutorial and reference the paper sited above as appropriate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Also, U expansion expression is missing $\Delta t$ terms.</p><p> </p>